### DISCUSSION TITLE
Search Scope of Result Source ID "LocalSharePointResults"

### DISCUSSION THREAD

**Discussion by tatsutanaka** [microsoft-search/pnp-modern-search#3991] [Q&A] [answered]
I have a question regarding the PnP Modern Search Web Parts.

When setting the Result Source ID to "LocalSharePointResults" in the properties of "PnP - Search Results," is my understanding correct that the search scope will cover the entire tenant?

**Comment by kasperbolarsen** [comment, marked as answer]
Correct,  it will cover all the content in the tenant that the current "user"/Identity has access to
